demos/source/aws_iot_demo_defender.c

```diff
@@ -0,0 +1,226 @@
+/*
+ * Amazon FreeRTOS V201908.00
```

Comment:
This is not the license header we use with the C SDK (it mentions Amazon FreeRTOS).

Could you copy the license header from one of the other C SDK files?